hack/generate-external-crds/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - github.com/kubernetes-sigs/gateway-api/config/crd/experimental?ref=v0.6.2
```

Comment:
In the future, we can just change the values in this list and re-run the script to update CRDs.